bots/config_plots.py

```diff
@@ -58,7 +58,7 @@
 ]
 PLT_WATERMARK = dict(
     source=(
-        "https://raw.githubusercontent.com/GamestonkTerminal/GamestonkTerminal/main/images"
+        "https://raw.githubusercontent.com/OpenBB-finance/OpenBBTerminal/main/images"
         "/gst_logo_lockup_rGreen_with_letters_only.png"
```

Comment:
we might want to change this image to a new one

Reply:
Have asked @tehcoderer about that.